lib/engine/graph.rb

```diff
@@ -103,6 +106,16 @@ def compute(corporation, routes_only: false)
         end
       end
 
+      if @home_as_token
+        home_hexes = Array(corporation.coordinates).map { |h| @game.hex_by_id(h) }
```

Comment:
probably could be done with one loop
Array(cords).each do |hex_id|
  hex = @game.hex_by_id(hex_id)
  ...